bin/origami-image-set-tools.js

```diff
@@ -48,6 +48,24 @@ program
 		});
 	});
 
+// Command to purge images from Origami Image Service
+program
+	.command('purge')
+	.option('-s, --source-directory <dir>', 'The directory to look for source images in', process.env.IMAGESET_SOURCE_DIRECTORY)
```

Comment:
You'll need to add the `imageServiceApiKey` option, or the purge will error. Maybe alias as `IMAGE_SERVICE_API_KEY` as an environment variable too?

Reply:
Added 👌